pkgs/tools/misc/inav-blackbox-tools/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, cairo }:
+
+stdenv.mkDerivation rec {
+  pname = "inav-blackbox-tools";
+  version = "2021-04-22";
```

If using a git commit, the convention is "unstable-YYYY-MM-DD": https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming

```suggestion
  version = "unstable-2021-04-22";
```